src/model/MultiValueColumn.ts

```diff
@@ -0,0 +1,257 @@
+/**
+ * Created by bikramkawan on 24/11/2016.
+ */
+import * as d3 from 'd3';
+import ValueColumn from './ValueColumn';
+import Column from './Column';
+import {IBoxPlotColumn, IBoxPlotData} from './BoxPlotColumn';
+import {IValueColumnDesc} from './ValueColumn';
+
+
+export class CustomSortCalculation {
+  private readonly aVal: number[];
+  private readonly bVal: number[];
+
+  constructor(aVal, bVal) {
+    this.bVal = bVal;
+    this.aVal = aVal;
+  }
+
+  sum() {
+    return (d3.sum(this.aVal) - d3.sum(this.bVal));
+  }
+
+  min() {
+    return (d3.min(this.aVal) - d3.min(this.bVal));
+  }
+
+  max() {
+    return (d3.max(this.aVal) - d3.max(this.bVal));
+  }
+
+  mean() {
+    return (d3.mean(this.aVal) - d3.mean(this.bVal));
+  }
+
+  median() {
+    return (d3.median(this.aVal.sort(numSort))) - (d3.median(this.bVal.sort(numSort)));
```

Comment:
yes I am sure. you can check in getboxplotdata method where i have sorteddata using instance of data